src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -77,6 +77,7 @@
     "suggest": {
         "ext-apcu": "For best performance of the system caches",
         "symfony/console": "For using the console commands",
+        "symfony/dsn": "For using the configuring locks and cache with Dsn",
```

Comment:
I think this should be moved to "require", so that DSN works out of the box in fwb.

Reply:
at least must be in require-dev, so that tests can pass

Reply:
moved into `require` section